internal/testlib/git.go

```diff
@@ -20,9 +20,9 @@ func GitInit(tb testing.TB) {
 }
 
 // GitRemoteAdd adds the given url as remote.
-func GitRemoteAdd(tb testing.TB, url string) {
+func GitRemoteAdd(tb testing.TB, remote, url string) {
```

Comment:
maybe create another method ` GitRemoteAddWithName`  to avoid having to repeat ` origin`  in a lot of places? wdyt?

Reply:
of course, makes sense 🤝